### ISSUE TITLE
How will we publish this?

### ISSUE THREAD

**Issue by jswelker** [jswelker/library-web-guidelines#1]
Will we just leave it in wiki form? Not ideal. Can we have it stamped by LITA and have a PDF version hosted on the LITA website?

**Comment by michaelschofield** [comment]
I think we can bootstrap a simple github page that pushes these resources to the front end. Take a look at [Website Style Guide Resources](http://styleguides.io), which is just that.

**Comment by jswelker** [comment]
I agree that we could/should have a simple frontend page on this repo. The question I meant to ask is how can we publish this in a way that it's stamped by traditional library organization like ALA? I mentioned LITA because the LITA UX interest group has expressed some interest in sponsoring the project, so to speak. Since one of the big goals is to use this document to get buy in from non-tech librarians and admins, we probably want to have the ALA name on it somehow.

I am thinking something like we put this thing together on the web, and after all is said and done, we convert it to some legacy-friendly format like PDF that can be hosted in the LITA website, distributed via email, etc.

**Comment by michaelschofield** [comment]
> I am thinking something like we put this thing together on the web, and after all is said and done, we convert it to some legacy-friendly format like PDF that can be hosted in the LITA website, distributed via email, etc.

+1!

**Comment by ghost** [comment]
I think it's important for it to be supported by ALA/LITA, if we want to get wider acceptance in the library community. Possibly shopping it out at the next ALA conference?

**Comment by sciencex** [comment]
+1 to the legacy-friendly format like PDF - linking to said github page.
